respy/tests/random_model.py

```diff
@@ -163,6 +181,14 @@ def generate_random_model(
         "n_periods": np.random.randint(1, bound_constr["max_periods"]),
     }
 
+    if observables is not False:
```

Comment:
The if clause is not necessary. If there are no indices, the dict comprehension will return an empty dict by default.

Reply:
Then I however get a key error since the observables are not in the indices of the params df. 
Would you suggest to move observable_covariates out of the loop?

Reply:
Got it. Try params.get(„observables“, []).

Reply:
That seems to not work for indices of pd DataFrames. Am I wrong?

Reply:
Everything else I came up with that would do that seems to be at least as complicated. So I am not quite sure whether it is worth it.